eessi-2023.06-eb-4.8.0-system.yml

```diff
@@ -0,0 +1,5 @@
+easyconfigs:
+  - Java-11.0.18.eb:
```

Comment:
Use `Java-11.eb` here rather than a specific Java version, which will install a `Java/11` wrapper - see also https://docs.easybuild.io/wrapping-dependencies.

We always depend on `Java/11` rather than a specific subversion, so we can install in-place updates.